src/tasks/rebuild-web-api.js

```diff
@@ -295,10 +295,11 @@ module.exports = function rebuildWebApi(functionName, functionVersion, restApiId
 				}
 				if (additionalErrors()) {
 					additionalErrors().forEach(function (additionalError) {
+						var additionalErrorConfig = additionalError.toConfig();
```

Comment:
you can override the Errors https://github.com/phips28/claudia-api-errors#override-error-classes
There you can set a different output template.
what else would you like to configure for an error?
---
so we should call the `toConfig()` function directly for every function to make the chaching work?
```
additionalErrors: [
  new ApiErrors.BadRequest().toConfig(),
],
```